docx/oxml/text/run.py

```diff
@@ -73,6 +85,30 @@ def clear_content(self):
         for child in content_child_elms:
             self.remove(child)
 
+    @property
+    def footnote_reference_ids(self) -> (list[int]|None):
+        """
+        Return all footnote reference ids (``<w:footnoteReference>``), or |None| if not present.
+        """
+        references = []
+        for child in self:
+            if child.tag == qn('w:footnoteReference'):
+               references.append(child.id)
+        if references == []:
+            references = None
+        return references
+
+    def increment_footnote_reference_id(self):
```

Comment:
Would it make sense to rename this method to something like:
`increment_containing_footnote_references_ids`?

Reply:
Yes, it makes sense